index.html

```diff
@@ -1,279 +1,292 @@
+
+            <nav>
+                <ul class="my-menu-nav">
+                    <li class="menu-expand"><a href="#">Portfolio</a></li>
+                    <li class="menu-expand"><a href="#">About</a></li>
+                    <li class="menu-expand"><a href="#">Contact</a></li>
+                 </ul>
+               <div class="my-menu">
+                  <div></div>
+                  <div></div>
+                  <div></div>
+               </div>
+            </nav>
+
+         </header>
+         <section class="headline">
+            <div class="head-items">
+            <h1 class="head1"> I’m George<br>
+               Glad to see you!
+            </h1>
+            <br>
             <p class="primary">I’m a software developer! I can help you
-                build a product , feature or website Look
-                 through some of my work and 
-                 experience! If you like what you see and
-                  have a project you need coded, don’t 
-                  hestiate to contact me.</p>
-                  <br>
-    
-        <a href="#"class="connect">LET’S CONNECT </a>
-        </div>
-            <br><br><br>
-            
-            <div class="icons" > 
-                <ul class="socials-2">                
-                    <li> <a href="https://twitter.com/geok8376"><img src="img/Vectortwit.png" class="twit"></a> </li>
-                    <li> <a href="https://www.linkedin.com/in/george-kinyanjui-030468149/"><img src="img/Linkedin icon.png"  alt="linkedin"> </a>  </li>
-                    <li> <a href="#"><img src="img/Vectorm.png" alt="m"></a> </li>
-                    <li> <a href="https://github.com/george827"><img src="img/Vectorgit.png" alt="git"></a> </li>
-                    <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
-                </ul>
+               build a product , feature or website Look
+               through some of my work and 
+               experience! If you like what you see and
+               have a project you need coded, don’t 
+               hestiate to contact me.
+            </p>
+            <a href="#"class="connect">LET’S CONNECT </a>
+            <br><br>
+            <div class="icons" >
+               <ul class="socials-2">
+                  <li> <a href="https://twitter.com/geok8376"><img src="img/Vectortwit.png" class="twit"></a> </li>
+                  <li> <a href="https://www.linkedin.com/in/george-kinyanjui-030468149/"><img src="img/Linkedin icon.png"  alt="linkedin"> </a>  </li>
+                  <li> <a href="#"><img src="img/Vectorm.png" alt="m"></a> </li>
+                  <li> <a href="https://github.com/george827"><img src="img/Vectorgit.png" alt="git"></a> </li>
+                  <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
+               </ul>
             </div>
-        </section>
-    
-    </article>
-
-    <section id="main-works">
```

Comment:
Please make sure to put some space between two sections.

![image](https://user-images.githubusercontent.com/34779928/194700584-18fc80d5-e40a-41ef-86b6-f10c6a5d7dde.png)